app/client/jsx/FlaskInterface.jsx

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios'
```

Comment:
apologies for nitpick, but could we rename so as not to bind to a specific backend framework? could we call this WebApi, or something else generic?

Reply:
as in, not Flask